### DISCUSSION TITLE
Simplified Embedded Rust: A Comprehensive Guide to Embedded Rust Development · Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#29] [General]
# Simplified Embedded Rust: A Comprehensive Guide to Embedded Rust Development · Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/books-simplified-embedded-rust/

**Comment by suda-morris** [comment]
🦀🦀🦀

**Comment by MabezDev** [comment]
:rocket:

**Comment by bjoernQ** [comment]
🦀🚀
